src/store/useCategoryStore.ts

```diff
@@ -22,6 +22,8 @@ const useCategoryStore = storeSupport<CategoryInfo>(
       return get().categoryList;
     },
     transformCategoryTagList: (categoryIds: number[]) => {
+      if (get().categoryList.length === 0) return [];
```

Comment:
```suggestion
if (!get().categoryList.length) return [];
```
P3: 숫자 리터럴 0 이 false라 이렇게 쓰셔도 괜찮을 것 같습니다!

Reply:
이전에 다른 곳에서 `!get().categoryList.length` 식으로 사용했던 적이 있었는데, `!` 사용이 간결하긴 하지만 length의 특성상 명시적으로 숫자를 표기하는 것이 코드의 의도를 명확하게 파악할 수 있어 가독성 측면에서 좋다는 피드백을 받은 적이 있습니다. 저도 그 피드백에 어느 정도 동의합니다!  
이런 부분은 커스텀 util 함수를 생성해서 리팩토링 해봐도 좋을 것 같습니다🙂

Reply:
설득력 있는 의견 같습니다! 개인적으로는 `!get().categoryList.length` 정도도 괜찮지 않나 싶긴 하지만 일단 유지하셔도 될 것 같습니다.